Neos.Media.Browser/Resources/Private/Templates/Asset/Show.html

```diff
@@ -85,9 +97,16 @@
 <f:section name="ContentImage">
     <label>{neos:backend.translate(id: 'preview', package: 'Neos.Media.Browser')}</label>
     <div class="neos-preview-image">
-        <a href="{assetProxy.originalUri}" target="_blank">
-            <img src="{assetProxy.previewUri}" class="img-polaroid" alt="{assetProxy.label}"/>
-        </a>
+        <f:if condition="{assetContainsMaliciousContent}">
+            <f:then>
+                <img src="{assetProxy.previewUri}" class="img-polaroid" alt="{assetProxy.label}"/>
```

Comment:
Are we sure the `previewUri` does never point to the potentially malicious content? 🤔

Reply:
See above.